clarify proposal author expectations at each phase

shelbeeee opened the pull request:
I have noticed the process/expectations for proposals authors is not clear enough in the existing governance manual. I have made a few suggestions here, but this PR will be a work in progress as I create more templates and resources for proposal authors over the coming weeks.

shelbeeee committed:
clarify proposal author expectations at each phase

I have noticed the process/expectations for proposals authors is not clear enough in the existing governance manual. I have made a few suggestions here, but this PR will be a work in progress as I create more templates and resources for proposal authors over the coming weeks.

abbey-titcomb commented on manual.md:
```diff
@@ -49,13 +49,17 @@ RAD Governance Proposals (RGPs) must be formally labeled with the phase of the p
 A RGP must first complete **an off-chain review process** before it can be submitted on-chain. Having a public off-chain review process allows for proposals to be reviewed, discussed, and improved by the Radworks community before reaching an on-chain vote. This process includes a *Discussion* and a *Formal Review*. These steps help provide on-chain voters with more context on community sentiment  ...
 
-During _Formal Review_, each proposal is required to publish a Snapshot poll to gauge consensus on whether it should proceed to an on-chain vote. This is the last chance for the community to propose changes and give feedback on the proposal before it moves into an on-chain vote.
+The *Discussion* phase is the primary time for the community to review and provide feedback on proposals. Proposal authors should try to include as much detail as possible in this first draft of their proposals. Proposal authors can use these templates to create their proposals, and should post them under [Proposal Discussions](https://community.radworks.org/c/governance/discussions/19) on the fo ...
```
should "these templates" link somewhere?

abbey-titcomb commented on manual.md:
```diff
@@ -49,13 +49,17 @@ RAD Governance Proposals (RGPs) must be formally labeled with the phase of the p
+The *Discussion* phase is the primary time for the community to review and provide feedback on proposals. Proposal authors should try to include as much detail as possible in this first draft of their proposals. Proposal authors can use these templates to create their proposals, and should post them under [Proposal Discussions](https://community.radworks.org/c/governance/discussions/19) on the fo ...
 
-To initiate a Snapshot poll for a Formal Review, you must be added as an 'author' in our Snapshot space. This measure is in place to prevent spam proposals. Please reach out to Shelby (shelb_ee#9785 on Discord) with the Ethereum address of the party posting your proposal. All Snapshot Polls must begin on **5:00pm GMT+2** the **third Monday of the month** and end at **5:00pm GMT+2** on the **follo ...
+_Formal Review_ is the last chance for the community to propose changes and give feedback on the proposal before it moves to an on-chain vote. During _Formal Review_, proposal authors are required to publish an updated version of their proposal on the forum taking into account feedback from the *Discussion* phase. If applicable, the proposal must be accompanied by executable on-chain code for the ...
```
```suggestion
_Formal Review_ is the last chance for the community to propose changes and give feedback on the proposal before it moves to an on-chain vote. During _Formal Review_, proposal authors are required to publish an new version of their proposal on the forum that takes feedback from the *Discussion* phase into account. If applicable, the proposal must be accompanied by a draft of the executable proposal code for the community to review. See this template [INSERT LINK WHEN READY] for how the code should be presented. Proposals in this stage should be posted under the [Active Proposals](https://community.radworks.org/c/governance/proposals/18) category on the forum. 
```

abbey-titcomb commented on manual.md:
```diff
@@ -49,13 +49,17 @@ RAD Governance Proposals (RGPs) must be formally labeled with the phase of the p
-To initiate a Snapshot poll for a Formal Review, you must be added as an 'author' in our Snapshot space. This measure is in place to prevent spam proposals. Please reach out to Shelby (shelb_ee#9785 on Discord) with the Ethereum address of the party posting your proposal. All Snapshot Polls must begin on **5:00pm GMT+2** the **third Monday of the month** and end at **5:00pm GMT+2** on the **follo ...
+_Formal Review_ is the last chance for the community to propose changes and give feedback on the proposal before it moves to an on-chain vote. During _Formal Review_, proposal authors are required to publish an updated version of their proposal on the forum taking into account feedback from the *Discussion* phase. If applicable, the proposal must be accompanied by executable on-chain code for the ...
+
+Each proposal is required to publish a Snapshot poll to gauge consensus on whether it should proceed to an on-chain vote. To initiate a Snapshot poll for a Formal Review, you must be added as an 'author' in our Snapshot space. This measure is in place to prevent spam proposals. Please reach out to Shelby (shelb_ee#9785 on Discord) with the Ethereum address of the party posting your proposal. All  ...
```
This is a lot of text to have in a paragraph. Perhaps we list it as a set of steps instead?

abbey-titcomb commented on manual.md:
```diff
@@ -49,13 +49,17 @@ RAD Governance Proposals (RGPs) must be formally labeled with the phase of the p
 **Submission**
 
-Submitting a proposal on-chain is the final step in the governance process. The proposal should incorporate feedback from the _Discussion_ and _Formal Review_ phases and must be accompanied by executable on-chain code. An RGP can be submitted to on-chain governance by anyone who has an amount of $RAD greater than or equal to **1% of the total $RAD supply** delegated to their address (i.e ≥ 1M $RA ...
+Submitting a proposal on-chain is the final step in the governance process. If proposals pass Formal Review, proposal authors are required to post an updated version of their proposal on the forum and should incorporate feedback from the _Discussion_ and _Formal Review_ phases. The proposal should inlcude links to both the *Discsussion* and the *Formal Review* versions of the proposal. The propos ...
```
Same feedback for this. Maybe its not a list of steps, but a "checklist" or something?

abbey-titcomb reviewed (changes requested):
added some comments.

shelbeeee committed:
Update manual.md

Co-authored-by: Abbey Titcomb <abbeytitcomb@gmail.com>

shelbeeee committed:
separated & clarified phases of gov process

separated & clarified phases of gov process, added lists for proposal authors for clarity based on Abbey's feedback

abbey-titcomb commented on manual.md:
```diff
@@ -45,17 +45,38 @@ The primary tools for Radworks governance are:
+
+**Formal Review**
+
+The _Formal Review_ phase serves as the final review step in the governance process before proposals are submitted on-chain. Proposals in this phase incorporate feedback from the _Discussion_ phase, and are accompanied by an off-chain Snapshot poll to gauge consensus for their advancement. All Snapshot Polls must begin on **5:00pm GMT+2** the **third Monday of the month** and end at **5:00pm GMT+ ...
```
```suggestion
The _Formal Review_ phase serves as the final review step in the governance process before proposals are submitted on-chain. Proposals in this phase incorporate feedback from the _Discussion_ phase, and are accompanied by an off-chain Snapshot poll to gauge consensus for their advancement. All Snapshot polls must begin on **5:00pm GMT+2** the **third Monday of the month** and end at **5:00pm GMT+2** on the **following Monday**. Each Snapshot poll should have three voting options: `Yes` `No` and `Abstain`. 
```

Let's de-capitalize "Polls"

abbey-titcomb commented on manual.md:
```diff
@@ -45,17 +45,38 @@ The primary tools for Radworks governance are:
+The _Formal Review_ phase serves as the final review step in the governance process before proposals are submitted on-chain. Proposals in this phase incorporate feedback from the _Discussion_ phase, and are accompanied by an off-chain Snapshot poll to gauge consensus for their advancement. All Snapshot Polls must begin on **5:00pm GMT+2** the **third Monday of the month** and end at **5:00pm GMT+ ...
+
+During _Formal Review_, proposal authors are required to:
+- Publish an new version of their proposal on the forum that takes feedback from the *Discussion* phase into account. Proposals in this stage should be posted under the [Active Proposals](https://community.radworks.org/c/governance/proposals/18) category on the forum. 
```
```suggestion
- Publish an new version of their proposal on the forum that incorporates feedback from the *Discussion* phase. Proposals in this stage should be posted under the [Active Proposals](https://community.radworks.org/c/governance/proposals/18) category on the forum. 
```

abbey-titcomb commented on manual.md:
```diff
@@ -45,17 +45,38 @@ The primary tools for Radworks governance are:
+
+During _Formal Review_, proposal authors are required to:
+- Publish an new version of their proposal on the forum that takes feedback from the *Discussion* phase into account. Proposals in this stage should be posted under the [Active Proposals](https://community.radworks.org/c/governance/proposals/18) category on the forum. 
+- If applicable, inlcude a draft of the executable proposal code for the community to review. See this template [INSERT LINK WHEN READY] for how the code should be presented. 
```
What link are you inserting in the [INSERT LINK WHEN READY]?

abbey-titcomb commented on manual.md:
```diff
@@ -45,17 +45,38 @@ The primary tools for Radworks governance are:
+For Submission, proposal authors are required to: 
+- Post an updated version of their proposal on the forum that incorporates feedback from the _Discussion_ and _Formal Review_ phases.
+- Inlcude links to both the *Discsussion* and the *Formal Review* versions of the proposal in the final Submission post.
+- Include any executable on-chain code in the Submission post. Please add or update the corresponding proposal code in the [proposal code archive](https://github.com/radicle-foundation/radworks-governance/tree/main/proposals) using this template [INSERT LINK WHEN READY] if not done so already. This version of the proposal should also be posted under the [Active Proposals](https://community.radwor ...
```
Maybe include a note about how one should add the code to the proposal code archive, aka by opening a PR.

abbey-titcomb commented on manual.md:
```diff
@@ -45,17 +45,38 @@ The primary tools for Radworks governance are:
+- Inlcude links to both the *Discsussion* and the *Formal Review* versions of the proposal in the final Submission post.
+- Include any executable on-chain code in the Submission post. Please add or update the corresponding proposal code in the [proposal code archive](https://github.com/radicle-foundation/radworks-governance/tree/main/proposals) using this template [INSERT LINK WHEN READY] if not done so already. This version of the proposal should also be posted under the [Active Proposals](https://community.radwor ...
+
+If a proposal does not pass _Submission_, it can be resubmitted in the next governance cycle. The new draft should take into account feedback from the community.
```
Anything else it should include? Maybe a specific "Resubmission" tag, or a specific callout at the top about why its being resubmitted and what changes are being made?

abbey-titcomb reviewed (approved):
add some comments and suggested some changes, but looks good to me!

shelbeeee committed:
Update manual.md

Co-authored-by: Abbey Titcomb <abbeytitcomb@gmail.com>

shelbeeee committed:
Update manual.md

Co-authored-by: Abbey Titcomb <abbeytitcomb@gmail.com>

shelbeeee committed:
Update manual.md

abbey-titcomb reviewed (approved):
lgtm!
